v-next/hardhat-keystore/src/internal/type-extensions.ts

```diff
@@ -0,0 +1,9 @@
+import "@ignored/hardhat-vnext/types/config";
+
+declare module "@ignored/hardhat-vnext/types/config" {
+  export interface HardhatConfig {
```

Comment:
The intent here is to expand the Hardhat config but not allow users to specify a file path to a keystore.